specs/agents/tracing-instrumentation-db.md

```diff
@@ -117,11 +117,15 @@ The following fields are relevant for database and datastore spans. Where possib
 |`action`| `request` |
 | __**context.db._**__  |<hr/>|<hr/>|
 |`_.instance`| e.g. `my-cluster` | [Cluster name](#cluster-name), if available.
-|`_.statement`| e.g. <pre lang="json">{"query": {"match": {"user.id": "kimchy"}}}</pre> | For Elasticsearch search-type queries, the request body may be recorded. Alternatively, if a query is specified in HTTP query parameters, that may be used instead. If the body is gzip-encoded, the body should be decoded first.|
+|`_.statement`| e.g. <pre lang="json">{"query": {"match": {"user.id": "kimchy"}}}</pre> | For Elasticsearch search-type queries, the request body SHOULD be recorded according to the `elasticsearch_capture_body_urls` option (see below). If the body is gzip-encoded, the body should be decoded first.|
```

Comment:
Is the second "should" (... the body ...)  intentionally non-normative?

Reply:
```suggestion
|`_.statement`| e.g. <pre lang="json">{"query": {"match": {"user.id": "kimchy"}}}</pre> | For Elasticsearch search-type queries, the request body SHOULD be recorded according to the `elasticsearch_capture_body_urls` option (see below). If the body is gzip-encoded, the body MUST be decoded first.|
```

Good catch. I think we can change it to MUST as storing a non-decoded body would be pretty useless.